### ISSUE TITLE
Training hyperparameters？

### ISSUE THREAD

**Issue by jianghaojun** [Alibaba-MIIL/STAM#2]
Quiet promising work which shows the great potential of Video Transformer. Looking forward to the training code and details about training hyperparameters!

**Comment by giladsharir** [comment]
thanks for your interest. 
The model is trained with batch size 8 for 100 epochs, with initial learning rate 0.001 and weight decay 1e-4.,  with cosine decay learning rate scheduler , and learning rate warm-up.  Trained on 8v100 server.
